crates/apollo-mcp-server/src/main.rs

```diff
@@ -164,6 +169,11 @@ async fn main() -> anyhow::Result<()> {
         OperationSource::from(ManifestSource::LocalHotReload(vec![manifest]))
     } else if !args.operations.is_empty() {
         OperationSource::from(args.operations)
+    } else if let Some(collection_id) = args.collection {
+        OperationSource::from(CollectionSource {
+            collection_id,
+            platform_api_config: platform_api_config()?,
+        })
```

Comment:
Can't we just this without implementing the `From` trait? This should be a dumb question... 😂

```suggestion
        OperationSource::Collection(CollectionSource {
            collection_id,
            platform_api_config: platform_api_config()?,
        })
```